libvuln/libvuln.go

```diff
@@ -139,9 +142,8 @@ func (l *Libvuln) GC(ctx context.Context) (int64, error) {
 // and stale vulnerabilites are removed in accordance with the UpdateRetention
 // value.
 //
-// GCFull may return an error accompanied by its other two return values;
-// the number of oustanding update operations to remove and the number of
-// vulnerabilities deleted.
+// GCFull may return an error accompanied by its other return value;
```

Comment:
```suggestion
// GCFull may return an error accompanied by its other return value,
```

The second clause is not a full sentence, so a semicolon is not appropriate.